torch_xla/csrc/ops/std_mean.cpp

```diff
@@ -0,0 +1,76 @@
+#include "torch_xla/csrc/ops/std_mean.h"
+
+#include "absl/strings/str_join.h"
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+#include "torch_xla/csrc/reduction.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::XlaOp LowerStd(xla::XlaOp input,
+                    const std::vector<xla::int64>& dimensions,
+                    bool keep_reduced_dimensions,
+                    xla::int64 correction) {
+  return BuildStdDeviation(input, dimensions, keep_reduced_dimensions, correction);
+}
+
+xla::XlaOp LowerMean(xla::XlaOp input,
+                     const std::vector<xla::int64>& dimensions,
+                     bool keep_reduced_dimensions) {
+  return BuildMean(input, dimensions, keep_reduced_dimensions);
+}
+
+xla::Shape NodeOutputShape(const Value& input,
+                           std::vector<xla::int64>& dimensions,
+                           bool keep_reduced_dimensions,
+                           xla::int64 correction) {
+  auto lower_for_shape_fn_std_mean = [&](absl::Span<const xla::XlaOp> operands) -> xla::XlaOp {
+    auto std = LowerStd(operands[0], dimensions, keep_reduced_dimensions, correction);
```

Comment:
Let's try not to use auto when we can 😄 , I think in general we only use it when the type name is super long and complicated. In this case we can just use `xla::XlaOp`.

Reply:
Done.